Asterix/optics/propagation_functions.py

```diff
@@ -567,7 +567,8 @@ def butterworth_circle(dim, size_filter, order=5, xshift=0, yshift=0):
     return butterworth
 
 
-def prop_fpm_regional_sampling(pup, fpm, nbres=np.array([0.1, 5, 50, 100]), samp_outer=2, filter_order=15, alpha=1.5):
+def prop_fpm_regional_sampling(pup, fpm, nbres=np.array([0.1, 5, 50, 100]), shift=(0, 0), samp_outer=2, filter_order=15,
```

Comment:
Is there a reason why you want one argument per line? I will do it, I was just curious since it stretches the code massively because Asterix has a lot of functions that each have a large number of arguments. And the average module length in Asterix is already very long.

Reply:
because my autoformater will always do it by default and if you don't do it, It will be done by my formater automatically the next time I format this file. Either the line fit in less than 121 and then on a single line or all parameter need to be on a column.

Reply:
ok